src/Data/Nat/Properties.agda

```diff
@@ -2237,6 +2237,24 @@ _>‴?_ = flip _<‴?_
 ≤‴⇒≤ : _≤‴_ ⇒ _≤_
 ≤‴⇒≤ = ≤″⇒≤ ∘ ≤‴⇒≤″
 
+≤‴-irrelevant : Irrelevant {A = ℕ} _≤‴_
+≤‴-irrelevant ≤‴-refl q = lemma q refl
+  where
+  lemma : ∀ {m n} → (p : m ≤‴ n) → (e : m ≡ n) → subst (m ≤‴_) e ≤‴-refl ≡ p
+  lemma ≤‴-refl       e rewrite ≡-irrelevant e refl = refl
+  lemma (≤‴-step m<m) refl = contradiction (≤‴⇒≤ m<m) (<-irrefl refl)
+≤‴-irrelevant (≤‴-step x<x) ≤‴-refl = contradiction (≤‴⇒≤ x<x) (<-irrefl refl)
```

Comment:
These two clauses suggest a refactoring to abstract out an appeal to `Irreflexive _≡_ _≤‴_` as follows:
```suggestion
  lemma (≤‴-step m<m) e with () <‴-irrefl e m<m
≤‴-irrelevant (≤‴-step m<m) ≤‴-refl  with () <‴-irrefl refl m<m
```
with a prior definition (before L2240) as follows:
```agda
<‴-irrefl : Irreflexive _≡_ _<‴_
<‴-irrefl eq = <-irrefl eq ∘ ≤‴⇒≤
```